ssl/ssl_ciph.c

```diff
@@ -2220,16 +2223,19 @@ int ssl_cipher_get_overhead(const SSL_CIPHER *c, size_t *mac_overhead,
 
             in = 1; /* padding length byte */
             out = EVP_CIPHER_get_iv_length(e_ciph);
+            if (out < 0) {
```

Comment:
and here also

Reply:
Thanks. I have submited a new commit to fix both of them.